hi-heavens/theVaultPB
# Test: Added Jest tests for the routes

### What does this PR do?
This PR adds comprehensive Jest tests for the three main endpoints of the application: Create Account, Get Account by Account Number, and Get All Accounts. The tests ensure the correctness and robustness of these endpoints.

### Description of the task completed
Jest test cases have been added to cover all aspects of the application's main endpoints. Each test case simulates various scenarios and input data to validate the behavior of the endpoints. The tests verify successful account creation, fetching an account by its account number, and retrieving a list of all accounts.

### How should this be manually tested?
1. Run the Jest test suite to execute all the test cases:
```bash
npm test
```
or
```bash
npm test-watch
```
2. Observe the test output to ensure that all test cases pass without errors.
